config/boards/odroidn2.conf

```diff
@@ -1,12 +1,14 @@
 # Amlogic S922X hexa core 2GB/4GB RAM SoC 1.8-2.4Ghz eMMC GBE USB3 SPI RTC
 BOARD_NAME="Odroid N2"
 BOARDFAMILY="meson-g12b"
-BOOTCONFIG="odroidn2_config"
-CPUMIN=1000000
-CPUMAX=2400000
+BOOTCONFIG="odroid-n2_defconfig"
 MODULES_LEGACY="media_clock firmware #decoder_common #stream_input #amvdec_mh264 #amvdec_h264 #amvdec_h264mvc #amvdec_h265 #amvdec_mmjpeg #amvdec_mpeg12 #amvdec_mmpeg4 #amvdec_mpeg4 #amvdec_vc1 #amvdec_vp9"
 MODULES_BLACKLIST_LEGACY="spidev spi_gpio spi_bitbang"
 KERNEL_TARGET="legacy,current,edge"
```

Comment:
removing "legacy" if we can't boot it successfully. but we don't remove meson64-legacy since it works with C4 / HC4. Whoever wants to use that strange combination.

Reply:
Yeah, this PR inadvertently also fixes HC4 legacy and SATA due to hardcoded dtb bugfix. I remember forum posts about it...